web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala

```diff
@@ -95,23 +96,27 @@ import scala.util.{Failure, Success}
         )
     )
 
-    val error = formData.firstValidationError.map { text =>
-      val emailNotVerified = "The email is not verified, check your spam folder if you don't see the email."
+    def resendVerifyEmailButton(text: String) = {
+      val emailNotVerified = ErrorMessages.emailNotVerified
+
+      Fragment(text match {
+        case `emailNotVerified` =>
+          mui
+            .Button(texts.resendEmail)
+            .variant(muiStrings.text)
+            .color(muiStrings.primary)
+            .onClick(_ => history.push("/resend-verify-email"))
+        case _ => None
+      })
+    }
 
+    val error = formData.firstValidationError.map { text =>
```

Comment:
I think the variable name of `text` should be more explicit, it should tell us that it's an error not a simple text
For example:
`error`,  `errorMessage`, `errorText`, etc